### PR TITLE
Update IoTDB nifi process doc

### PR DESCRIPTION
**Pull request by lizhizhou**
## Description
Update IoTDB nifi process doc. 
Include 
PutIoTDBRecord
QueryIoTDBRecord
### Content3 ...

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

**Comment by HTHou** [comment]
@xuanronaldo PTAL

**Review by xuanronaldo — APPROVED** [review]
looks good for me

File: docs/UserGuide/Ecosystem-Integration/NiFi-IoTDB.md [hunk 1]
```diff
@@ -46,29 +46,30 @@ Apache NiFi includes the following capabilities:
...
 
 ### Inferred Schema of Flowfile
 
 There are a couple of rules about flowfile:
 
 1. The flowfile can be read by `Record Reader`.
-2. The schema of flowfile must contains a field `Time`, and it must be the first.
+2. The schema of flowfile must contain a field `Time`, and it must be the first.
 3. The data type of time must be `STRING` or `LONG`.
 4. Fields excepted time must start with `root.`.
```
Comment (xuanronaldo) [inline docs/UserGuide/Ecosystem-Integration/NiFi-IoTDB.md:74]: It seems like that the fields do not need to start with `root.`, if the `prefix` property is assigned. Besides, the relationship between `tsName` and field name is not  clarified.

Comment (lizhizhou) [reply docs/UserGuide/Ecosystem-Integration/NiFi-IoTDB.md:74]: Resolved

**Pull request merged**